testing/generate.js

```diff
@@ -53,13 +73,34 @@ function playwrightTest(appName, testNames, folderName) {
   const encodedFolderName = encodeURIComponent(folderName);
   const testAppName = appName.replace("'", "");
 
+  const beforeEachHook =
+`  test.beforeEach(async ({ page }) => {
+    try {
+      if (!fs.existsSync(resultsPath)) {
+        const app = new RetoolApplication(page, "${encodedAppName}", "${folderName ? encodedFolderName : ''}")
+        const results = await app.test()
+
+        fs.writeFileSync(resultsPath, results)
+      }
+    } catch(err) {
+      console.error(err)
+    }
```

Comment:
We shouldn't be eating any errors here. If the beforeEach function fails, we should let it throw and exception and have it reported by Playwright.